gqlparser.go

```diff
@@ -11,15 +11,7 @@ import (
 )
 
 func LoadSchema(str ...*ast.Source) (*ast.Schema, error) {
-	ast, err := validator.LoadSchema(append([]*ast.Source{validator.Prelude}, str...)...)
-	gqlErr, ok := err.(*gqlerror.Error)
-	if ok {
```

Comment:
This change made it obvious that `ok` is always true.

Reply:
That wasn't entirely accurate, but I felt like it was reasonable that it should be, so I changed the code `validator.LoadSchema` to call a convenience WrapIfUnwrapped.

The issue was that in order to avoid cyclical dependencies, we cannot produce gqlerrors from the ast package.